### DISCUSSION TITLE
Registration+Label Propagation with prior knowledge of labels

### DISCUSSION THREAD

**Discussion by saurabh-vaishampayan** [ANTsX/ANTs#1494] [Q&A]
Hello

My project involves a MultiATLAS based registration and label propagation. I have a prior on labels of the fixed image. How can I use this to aid the registration process (to improve performance)?

**Comment by ntustison** [comment]
It would help if you made some of the data available. Or at least gave more details with images to show what you're trying to do.  Also, please explain what you've already tried.
